src/oemof/solph/_models.py

```diff
@@ -441,3 +444,46 @@ def relax_problem(self):
         relaxer._apply_to(self)
 
         return self
+
+
+class Results:
```

Comment:
As this in not depending on `solph.Model`, I see this class going to processing.